AppCenterAuth/AppCenterAuth.xcodeproj/project.pbxproj

```diff
@@ -925,20 +933,27 @@
 			remoteRef = 04E9760C226A50BF007E6DD1 /* PBXContainerItemProxy */;
 			sourceTree = BUILT_PRODUCTS_DIR;
 		};
-		04E9760F226A50BF007E6DD1 /* libMSAL.a */ = {
+		04E9760F226A50BF007E6DD1 /* libMSAL (iOS Static Library).a */ = {
```

Comment:
I'm curious about these file name changes.  I don't think they should be present, there shouldn't be any reason to change them